app/views/page/homePage.scala.html

```diff
@@ -26,12 +26,13 @@
 *@
 
 @main("Dr. Elephant", "dashboard") {
-  <div class="jumbotron">
-    <h2>Hello there, I've been busy!</h2>
-    <p>I looked through <b>@numJobsAnalyzed</b> jobs today.<br>
-      About <b>@numJobsSevere</b> of them could use some tuning.<br>
-      About <b>@numJobsCritical</b> of them need some serious attention!
-    </p>
-  </div>
-  @results
+<div class="jumbotron">
+  <h2>Hello there, I've been busy!</h2>
+  <p>I've looked through <b>@("%,d".format(numJobsAnalyzedLastDay))</b> jobs that finished in the last 24 hours.<br>
```

Comment:
Cool. I mistook the number for the jobs Dr. Elephant analyzed. Long time back, we used to track Dr. Elephant's analysis time of a job in the db and the above number in the dashboard would tell the exact number of jobs Dr. Elephant analyzed. But later we removed the analysis time as it wasn't necessary and replaced it with the finish time. We can now only group based on the finish time of the job and not the analysis time. 

Thanks for the clarification!